.circleci/config.yml

```diff
@@ -2,19 +2,20 @@
 
 version: 2
 references:
-  container_python36: &container_python36
+  container_python310: &container_python310
     docker:
-      - image: circleci/python:3.6.2
+      - image: cimg/python:3.10.11
       - name: localstack
-        image: localstack/localstack
-    working_dir: ~/project
+        image: localstack/localstack:2.0.2
+    working_directory: ~/repo
```

Comment:
What's the difference between `working_dir` and `working_directory`?

Reply:
https://app.circleci.com/pipelines/github/nasa/cumulus-process-py/128/workflows/4d7fccd6-ed13-439b-b7dc-49cda94e741c/jobs/1110
Why the `localstack/localstack:2.0.2` build is cancelled?

Reply:
> What's the difference between `working_dir` and `working_directory`?
working_dir appeared to be working but working_directory is what's documented online and also the keyword we're using elsewhere i.e. cma-python, so i've moved it to working_directory while debugging

Reply:
> Why the `localstack/localstack:2.0.2` build is cancelled?
I had to go double check this, but it's 'cancelled' automatically to keep the server running in background while other stuff goes
https://discuss.circleci.com/t/build-was-cancelled-in-database-containers/34411
according to this convo, this appears to be normal. it is how all of our other container steps work in other circle-ci pipelines